### PR TITLE
 Refresh

### PR DESCRIPTION
**Pull request by brad**
@orcasgit/orcas-developers Please review. I updated the tox matrix and added docstrings to all the models and `help_text`s to all the model fields.

### CODE REVIEW

File: misfitapp/models.py [hunk 1]
```diff
@@ -72,24 +72,38 @@ def import_all_from_misfit(cls, misfit, uid):
...
+    https://build.misfit.com/docs/cloudapi/api_references#summary
+    """
+    user = models.ForeignKey(UserModel, help_text="The summary's user")
+    date = models.DateField(help_text='The date of the summary')
+    points = models.FloatField(help_text='Points for the day')
+    steps = models.IntegerField(help_text='Steps for the day')
+    calories = models.FloatField(help_text='Calories for the day')
+    activity_calories = models.FloatField(
+        help_text='Activity calories for the day')
+    distance = models.FloatField(help_text='Distance travelled during the day')
```
Comment (grokcode) [inline misfitapp/models.py]: @brad It might be helpful to put the units here.

File: misfitapp/models.py [hunk 2]
```diff
@@ -164,13 +192,26 @@ def import_from_misfit(cls, misfit, uid, object_id=None):
...
+    https://build.misfit.com/docs/cloudapi/api_references#device
+    """
+    id = models.CharField(
+        max_length=MAX_KEY_LEN,
+        primary_key=True,
+        help_text='The device ID assigned by Misfit')
+    user = models.ForeignKey(UserModel, help_text="The device's user")
+    device_type = models.CharField(
+        max_length=64,
+        help_text='The device type. At this time, will always be "shine"')
```
Comment (grokcode) [inline misfitapp/models.py]: @brad I know Misfit's docs say this, but there are some other values that Misfit will send besides `shine`. #9

Comment (brad) [inline misfitapp/models.py]: @grokcode Oh, I forgot, thanks!

File: misfitapp/models.py (hunk 1 above)
Comment (brad) [inline misfitapp/models.py]: @grokcode Good call, the documentation doesn't specify but it's probably miles. That would fit well with the examples and it looks like distance on other API end points come in miles.

**Comment by grokcode** [comment]
LGTM! Thanks for this update! :shipit:

**Pull request merged**